packages/models-library/src/models_library/api_schemas_webserver/projects_nodes.py

```diff
@@ -121,7 +121,11 @@ class Config:
         }
 
 
-class NodeRetrieve(InputSchemaWithoutCameCase):
+class NodeOutputs(InputSchemaWithoutCamelCase):
+    outputs: dict[str, Any]
```

Comment:
Just question as last time about the structure of this field, whether we want to improve with some type hints or it is not convenient now.

Reply:
Thanks. I investigated this together with @pcrespov and we concluded we cannot restrict it further. This could really be any json serializable dict whose keys are strings